sw/device/silicon_creator/rom/rom.c

```diff
@@ -259,35 +272,42 @@ static rom_error_t rom_verify(const manifest_t *manifest,
     HARDENED_CHECK_EQ(sigverify_spx_en, kSigverifySpxDisabledOtp);
   }
 
+  /**
+   * Measure ROM_EXT and portions of manifest via SHA256 digest.
+   */
+  // Initialize ROM_EXT measurement in .static_critical with garbage.
   memset(boot_measurements.rom_ext.data, (int)rnd_uint32(),
          sizeof(boot_measurements.rom_ext.data));
-
+  // Add anti-rollback poisoning word to measurement.
   hmac_sha256_init();
   hmac_sha256_update(anti_rollback, anti_rollback_len);
   HARDENED_CHECK_GE(manifest->security_version,
                     boot_data.min_security_version_rom_ext);
-
-  // Hash usage constraints.
+  // Add manifest usage constraints to the measurement.
   manifest_usage_constraints_t usage_constraints_from_hw;
   sigverify_usage_constraints_get(manifest->usage_constraints.selector_bits,
                                   &usage_constraints_from_hw);
   hmac_sha256_update(&usage_constraints_from_hw,
                      sizeof(usage_constraints_from_hw));
-  // Hash the remaining part of the image.
+  // Add remaining part of manifest / ROM_EXT image to the measurement.
   manifest_digest_region_t digest_region = manifest_digest_region_get(manifest);
-
+  // Add remaining part of manifest / ROM_EXT image to the measurement.
   hmac_sha256_update(digest_region.start, digest_region.length);
-  // Verify signature
   hmac_digest_t act_digest;
   hmac_sha256_final(&act_digest);
-
+  // Copy the ROM_EXT measurement to the .static_critical section.
   static_assert(sizeof(boot_measurements.rom_ext) == sizeof(act_digest),
                 "Unexpected ROM_EXT digest size.");
   memcpy(&boot_measurements.rom_ext, &act_digest,
          sizeof(boot_measurements.rom_ext));
 
   CFI_FUNC_COUNTER_INCREMENT(rom_counters, kCfiRomVerify, 2);
-  // Swap the order of signature verifications randomly.
+
+  /**
+   * Verify the RSA/SPX+ signatures of ROM_EXT.
+   *
+   * We swap the order of signature verifications randomly.
+   */
```

Comment:
nit: In-code comments should follow the `// ..` format following the C/C++ style guide.

Reply:
fixed.